proto/ibc/core/client/v1/genesis.proto

```diff
@@ -18,8 +18,9 @@ message GenesisState {
   // metadata from each client
   repeated IdentifiedGenesisMetadata clients_metadata = 3 [(gogoproto.nullable) = false];
   Params                             params           = 4 [(gogoproto.nullable) = false];
-  // create localhost on initialization
-  bool create_localhost = 5;
+  // Deprecated: create_localhost has been deprecated.
```

Comment:
micro nit: seems like the deprecated option adds a similar comment to this so adding it here too might be on the side of redundancy :smile:

Reply:
If you (or somebody else) are not strongly opinionated, then I think I will just leave it. :)